packages/prisma/src/prisma/schema.prisma

```diff
@@ -104,7 +104,7 @@ enum QuestionType {
 }
 
 model Question {
-  id String @id @default(uuid()) @db.Uuid
+  id Int @id @default(autoincrement())
```

Comment:
Migrated everything that we do not expose for public access (e.g., a join link) to int ids for efficiency